style.css

```diff
@@ -92,6 +92,27 @@ body {
     margin-right: 8px;
   }
 
+  .top-navigation {
+    display: none;
+  }
+
+  #top-navigation {
+    margin: 34px 35px 0 24px;
+    transition: height 0.5s ease-out;
+    display: none;
+  }
+
+  #top-navigation a {
+    text-decoration: none;
+    color: #fff;
+    font-family: Roboto, sans-serif;
+    font-weight: 700;
+    font-size: 18px;
+    line-height: 133%;
+    margin: 0;
+    padding: 0 16px;
+  }
+
```

Comment:
- [ ] Great effort working on your mobile menu, Currently i notice your design is not the same with what we have in the Figma design. I would kindly suggest you make the height of the mobile menu cover the whole page when it is displayed. Also i would want you to look at the font size and be sure they are the same. You can check the id `top-navigation` and give it a `height: 100vh`. I believe this should work for the height issue.

| Figma Design 👇| Your design 👇|
| ------------- | ------------- |
| <img width="216" alt="orig-mob" src="https://github.com/erickma1/Module1_Portfolio_Setup/assets/78634841/1d85c792-a9a2-4f37-9774-35c53cc71df5">  | <img width="331" alt="mob-men" src="https://github.com/erickma1/Module1_Portfolio_Setup/assets/78634841/a4122a0e-97b5-476e-80da-02f876195f36">  |